proposal/09510/README.rst

```diff
@@ -0,0 +1,64 @@
+========================
+Encryption Acceleration 
+========================
+
+
+Basic Information
+========================
+Code repository: https://github.com/09510/Encryption-Acceleration
+
+Problem to solve
+=======================
+Message encryption is an important technology to protect information security. 
+However, if you want to ensure the encryption strength, the encryption/decryption takes a lot of time.
+Encryption Acceleration is a tool to accelerate encryption and decryption by GPU. 
```

Comment:
I will only evaluate the part of your project that works on an ordinary server with CPU.  It is OK for you to include code for GPU, but I will not take that part into consideration when grading.